chains/manager.go

```diff
@@ -579,10 +579,6 @@ func (m *manager) createAvalancheChain(
 	if err != nil {
 		return nil, err
 	}
-	blockBlocker, err := queue.NewWithMissing(blockBootstrappingDB, "block", ctx.Registerer)
```

Comment:
Do we want any of the metrics from in here?

Reply:
I'm going to say no - I don't think we need the ETA metric - and we no long have a cache here (because there are no reads!)